.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,27 @@
+name: e2e
+on:
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
+  schedule:
+  - cron: '0 0 * * *' # daily to pick up releases
```

Comment:
I think my preference would be to remove this cron schedule entirely as I don't think it adds a ton of value and just increases our overall actions cost. We can re-evaluate that preference once we get a more fleshed out repository that needs a more solidified CI/CD pipeline.

Reply:
Sounds like a plan, let's readdress if this becomes missed.